### DISCUSSION TITLE
Clear the current line

### DISCUSSION THREAD

**Discussion by zeitbit** [nushell/nushell#3744] [Q&A]
Hello, 

Is possible to configure the shell to use the key ESC to clear the current line (like Ctrl+U)?

Thank you.

**Comment by ghost** [comment]
In my experience, this configuration often requires customizing your terminal emulator.
So I believe you would want to bind your preferred keyboard shortcut to the nu command `clear`. 

What OS are you running? and terminal emulator?
For example, on a GNU/Linux system I often use Gnome Terminal [which has Ctrl + L bound](https://help.gnome.org/users/gnome-terminal/stable/adv-keyboard-shortcuts.html.en) to the reset functionality

**Reply by zeitbit** [reply]
I'm using nu in Windows 10 and the terminal PowerShell and in Sublime Text with the plugin Terminus.

**Comment by youk** [comment]
Add (or change existing binding)
```
{
    name: escape
    modifier: none
    keycode: escape
    mode: emacs
    event: { edit: clear }
}
```
to `keybindings` in `config.nu`.

See [Keybindings](https://www.nushell.sh/book/line_editor.html#keybindings) for reference.
